### DISCUSSION TITLE
Insecure access through ip address

### DISCUSSION THREAD

**Discussion by fernandomullerappmax** [gravitational/teleport#8849] [Q&A] [answered]
Is it possible to avoid Teleport to listen via ip address in Web UI?

Because i have TLS certificate to my fqdn and the connection is secure through HTTPS, but the access to the Web UI is unsecure through ip address because don't have a valid TLS certificate.

Any suggestions?

**Comment by webvictim** [comment, marked as answer]
Networking-wise, no, it isn't possible to stop Teleport from listening on an IP address. If this is a real concern, you could potentially run a reverse proxy nginx or Caddy in front of Teleport which could rewrite requests for the IP address to requests for the FQDN instead.
